test/collections.test.js

```diff
@@ -65,6 +65,20 @@ describe('payload collections', () => {
     expect(collection.conflictsOf(payload.uuid)).to.eql([manualResults]);
   });
 
+  it.skip('setting same element twice should not yeild duplicates', async () => {
```

Comment:
This test presently doesn't pass, and is a new one created to catch a pre-existing issue. I didn't want to modify this behavior in this PR, so marked it as skip for now.